src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -196,6 +196,12 @@ export const SearchBarDropdown = ({
     }
   }, [toggleOpen, hoveredIndex, totalSuggestions])
 
+  const hasVerifiedCollection = collections.some((collection) => collection.isVerified)
+  const hasVerifiedToken = tokens.some((token) => token.onDefaultList)
+  const showCollectionsFirst =
+    (isNFTPage && (hasVerifiedCollection || !hasVerifiedToken)) ||
```

Comment:
add `profile` to isNFTPage check

Reply:
gonna do in sep pr bc i think other places are broken too